book/website/collaboration/github-novice/github-novice-advanced.md

```diff
@@ -1,18 +1,26 @@
 (cl-github-novice-advanced)=
-# Advanced Github features
-
+# Advanced GitHub features
 ## More collaboration
-You can add collaborators to your project and learn how to use branches, pull requests and merges to aid collaborative work.
-Take a look at the next chapter on 'Collaborating on Github' for more information on how to do this.
+GitHub makes it easy to collaborate and create and manage teams.
+You can add collaborators to your organization or project and manage permissions using the Teams settings.
+Branches, pull requests and merges further facilitate streamlined, version-controlled, collaborative work.
+Take a look at the next chapter on {ref}`Maintainers and Reviewers on GitHub<cl-maintain-review>` for more information on how to use these features.
+
+## Turn your project into a web page
+Your GitHub repo can be turned into a web page for your project by enabling GitHub pages in the settings.
+You can customise the web pages using one of the templates in GitHub or you can choose your own [Jekyll](https://jekyllrb.com/) theme.
 
-## Try turning your project into a web page
-Your Github repo can be turned into a web page for your project by enabling Github pages in the settings.
-You can customise the web pages using one of the templates in Github or you can choose your own jekyll theme.
+This resource - [GitHub for collaborative documentation](https://cassgvp.github.io/github-for-collaborative-documentation/) - has a [step-by-step guide](https://cassgvp.github.io/github-for-collaborative-documentation/docs/tut/4-2-Make-your-Pages-site.html) of how to make GitHub pages.
 
-This resource - [Github for collaborative documentation](https://cassgvp.github.io/github-for-collaborative-documentation/) - has a step-by-step guide of how to make Github pages.
+## Link this repository with an online platform
 
-## Link this repository with an online platform such as Open Science Framework
-To make an open workflow linking all of the tools that you use, an online platform is needed.
-Open Science Framework enables researchers to add on many different tools such as Dropbox, Github and Google workspace.
+### Open Science Framework
+[Open Science Framework]() is an online platform that enables researchers to link many different tools - such as Dropbox, GitHub and Google workspace - to create a completely open workflow.
```

Comment:
```suggestion
[Open Science Framework](https://osf.io/) is an online platform that enables researchers to link many different tools - such as Dropbox, GitHub, and Google workspace - to create a completely open workflow.
```

Reply:
Thanks for catching all these typos, @da5nsy! I agree with your use of the Oxford comma, but The Turing Way doesn't have an official policy on them (unless that's changed recently!) other than consistency, so I was going with what was already there.

Reply:
Fair enough!